waiter/src/waiter/handler.clj

```diff
@@ -810,9 +810,10 @@
       (utils/clj->streaming-json-response {:details (->> ["autoscaler" "autoscaling-multiplexer" "codahale-reporters"
                                                           "ejection-expiry" "entitlement-manager" "fallback"
                                                           "gc-broken-services" "gc-services" "gc-transient-metrics" "interstitial"
-                                                          "jwt-auth-server" "kv-store" "launch-metrics" "leader" "local-usage"
-                                                          "maintainer" "router-metrics" "scheduler" "service-description-builder"
-                                                          "service-maintainer" "statsd" "token-watch-maintainer" "work-stealing"]
+                                                          "instance-tracker" "jwt-auth-server" "kv-store" "launch-metrics" "leader"
```

Comment:
lexicographic order, it can go on previous line and none of the other lines have to change.